routes/aliemcards/messages.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express';
+import { requireAuthentication } from '../../helpers/authentication';
+const router = express.Router();
+const CHANNEL_ID = '**********'; // #aliemcards
+
+export default function route(bot: Botkit.Bot): express.Router {
+
+    router.post('/contact-form', requireAuthentication, (req, res) => {
+        const { name, email, message } = JSON.parse(req.body.data.replace(/\r\n/g, '\\n'));
+        if (!name || !email || !message) return res.sendStatus(400);
+        bot.say({
```

Comment:
Make sure the message looks the way you want it here: https://api.slack.com/docs/messages/builder